src/rpc/rpc.cc

```diff
@@ -1185,6 +1185,18 @@ future<> server::connection::send_unknown_verb_reply(std::optional<rpc_clock_typ
       ).discard_result();
   }
 
+  void server::abort_connection(connection_id id) {
+      auto it = _conns.find(id);
+      if (it == _conns.end()) {
+          return;
+      }
+      try {
+          it->second->abort();
+      } catch (...) {
+          log_exception(*this, log_level::error, "fail to shutdown connection on user request", std::current_exception());
```

Comment:
@gleb-cloudius I wasn't sure about this catch - a similar catch is in `connection::stop()`. I think it makes sense - can the user do anything smart about `_fd.shutdown_input()` error?

Reply:
> can the user do anything smart about _fd.shutdown_input() error?

Not really.